src/vega.schema.ts

```diff
@@ -1084,10 +1096,11 @@ const VG_MARK_CONFIG_INDEX: Flag<keyof VgMarkConfig> = {
   font: 1,
   fontSize: 1,
   fontWeight: 1,
-  fontStyle: 1
+  fontStyle: 1,
+  href: 1,
+  cursor: 1,
```

Comment:
I'm not sure why previously list `cursor` as "vg channel that do not have mark config".  Can you test if cursor works below for sure?

Btw, does it really make sense to have `href` as a mark config too?

Reply:
Working on it

Reply:
```
{
  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
  "description": "A simple bar chart with embedded data.",
  "data": {
    "values": [
      {"a": "A","b": 28}, {"a": "B","b": 55}, {"a": "C","b": 43},
      {"a": "D","b": 91}, {"a": "E","b": 81}, {"a": "F","b": 53},
      {"a": "G","b": 19}, {"a": "H","b": 87}, {"a": "I","b": 52}
    ]
  },
  "mark": "bar",
  "encoding": {
    "x": {"field": "a", "type": "ordinal"},
    "y": {"field": "b", "type": "quantitative"}
  },
  "config": {
    "bar": {
      "cursor": "pointer"
    }
  }
}
```

works